python/sqlcommenter-python/setup.py

```diff
@@ -37,10 +37,10 @@ def read_file(filename):
     extras_require={
         'django': ['django >= 1.11'],
         'flask': ['flask'],
-        'psycopg2': ['psycopg2'],
+        'psycopg2': ['psycopg2 == 2.8.6'],
         'sqlalchemy': ['sqlalchemy'],
         'opencensus': ['opencensus'],
-        'opentelemetry': ["opentelemetry-api; python_version >= '3'"],
+        'opentelemetry': ["opentelemetry-api ~= 1.0; python_version >= '3'"],
```

Comment:
If we don't support python 2.7 anyway, I suppose we can clean this up
```suggestion
        'opentelemetry': ["opentelemetry-api ~= 1.0"],
```

Reply:
Great catch! Done elimination this part!